gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general-ng.component.html

```diff
@@ -0,0 +1,104 @@
+              >Application description is required.
+            </mat-error>
+            <mat-hint align="start">Provide a description of your application, what it does, ...</mat-hint>
+          </mat-form-field>
+
+          <mat-form-field class="details-card__header__info-inputs__second-row__domain-field">
+            <mat-label>Domain</mat-label>
+            <input formControlName="domain" matInput type="text" />
+            <mat-hint align="start">The domain your application will use, if relevant.</mat-hint>
+          </mat-form-field>
+        </div>
+      </div>
+      <div class="details-card__header__right-coll">
+        <div class="details-card__header__right-coll__media" [formGroup]="applicationImagesForm">
+          <gio-form-file-picker class="details-card__header__right-coll__media__picture" formControlName="picture" accept="image/*">
+            <gio-form-file-picker-label>Application picture</gio-form-file-picker-label>
+            <gio-form-file-picker-add-button class="details-card__header__right-coll__media__picture__btn">
+              <span class="details-card__header__right-coll__media__picture__btn__text"> Click here or drag an image </span>
+              <gio-avatar
+                class="details-card__header__right-coll__media__picture__btn__default-avatar"
+                size="114"
+                [name]="applicationDetails.value + ' ' + applicationDetails.value"
+              ></gio-avatar>
+            </gio-form-file-picker-add-button>
+            <gio-form-file-picker-empty>
+              <gio-avatar size="114" [name]="applicationDetails.value + ' ' + applicationDetails.value"></gio-avatar>
+            </gio-form-file-picker-empty>
+          </gio-form-file-picker>
+          <gio-form-file-picker class="details-card__header__right-coll__media__background" formControlName="background" accept="image/*">
+            <gio-form-file-picker-label>Application background</gio-form-file-picker-label>
+            <gio-form-file-picker-add-button class="details-card__header__right-coll__media__background__btn">
+              <span class="details-card__header__right-coll__media__background__btn__text"> Click here or drag an image </span>
+            </gio-form-file-picker-add-button>
+            <gio-form-file-picker-empty><span>No background defined</span></gio-form-file-picker-empty>
+          </gio-form-file-picker>
+        </div>
+        <div class="details-card__header__right-coll__info">
+          <dl class="gio-description-list">
+            <dt>Owner</dt>
+            <dd gioClipboardCopyWrapper [contentToCopy]="applicationDetails.owner.displayName">
+              {{ applicationDetails.owner.displayName }}
+            </dd>
+            <dt>Created</dt>
+            <dd>{{ applicationDetails.created_at | date : 'medium' }}</dd>
+            <dt>Type</dt>
+            <dd>{{ applicationDetails.type }}</dd>
```

Comment:
Note for later. it would be nice to display the displayable name , like in create pages and the id only if it's not found, no. 
I don't really know where to find it, but we can have a look on create pages code

Reply:
Okay